migrations/11_SCHEME-REP-Lock_Whitelisted_tokens.js

```diff
@@ -23,8 +25,12 @@ module.exports = async function (deployer) {
   const dxController = await DxController.deployed()
   const gnoToken = await GnoToken.deployed()
 
-  console.log('Deploy DxLockGno4Rep that inherits from LockingToken4Reputation')
-  const dxLockGno4Rep = await deployer.deploy(DxLockGno4Rep)
+
+  console.log('Deploy FixedPriceOracle for setting the prices for the tokens')
+  const fixedPriceOracle = await deployer.deploy(FixedPriceOracle)
```

Comment:
Why are you deploying `FixedPriceOracle` here and not using it anywhere? Will it be passed into `DxLockWhitelisted4Rep` when `LockWhitelisted4Rep` is ready?